### PR TITLE
Completed the input component

### PR DESCRIPTION
**Pull request by therajatg**

### CODE REVIEW

File: styles/component-styles/alert.css [hunk 1]
```diff
@@ -19,29 +6,29 @@
...
     border-radius: 1rem;
-    border: 2px solid var(--info-border);
+    border: 2px solid var(--info-border-text-color);
 }
 
 .alert-success{
-    background-color: var(--success-background);
+    background-color: var(--success-background-color);
     padding: 1rem;
     border-radius: 1rem;
```
Comment (ZuberDunge) [inline styles/component-styles/alert.css]: if you're giving padding and radius same to all alerts you can add a separate class for common styling

**Review by ZuberDunge — APPROVED** [review]
good to go.

File: styles/component-styles/alert.css (hunk 1 above)
Comment (therajatg) [reply styles/component-styles/alert.css]: Good observation Zuber. made a separate class for margin and padding.

**Review by ZuberDunge — APPROVED** [review]

**Comment by iamsaurav28** [comment]
looks good to merge go on

**Review by iamsaurav28 — APPROVED** [review]

**Pull request merged**